### PR TITLE
feat: use selection colors from theme for listbox

### PR DESCRIPTION
**Pull request by haxxmaxx**
## Motivation

Adding support for selection colors from theme, for the listbox. This means that chart overrides have the highest prio, then the theme, then the mui theme, the the hardcoded default values. new functionality is behind flag.

## Requirements checklist

- [x] Api specification
  - [x] Ran `yarn spec`
    - [x] No changes **_OR_** API changes has been formally approved
- [x] Unit/Component test coverage
- [x] Correct PR title for the changes (fix, chore, feat)

When build and tests have passed:

- [ ] Add code reviewers, for example @qlik-oss/nebula-core

### CODE REVIEW

**Comment by haxxmaxx** [comment]
#devbuild

File: apis/nucleus/src/components/listbox/ListBoxInline.jsx [hunk 1]
```diff
@@ -101,7 +102,8 @@ function ListBoxInline({ options, layout }) {
   const { translator, keyboardNavigation, themeApi, constraints } = useContext(InstanceContext);
 
   const { checkboxes = checkboxesOption } = layout || {};
-  const styles = getStyles({ app, themeApi, theme, components, checkboxes });
+  const themeSelectionColorsEnabled = flags.isEnabled('PS_22149_THEME_SELECTION_COLORS');
```
Comment (haxxmaxx) [inline apis/nucleus/src/components/listbox/ListBoxInline.jsx:105]: potentially we can get the flag value earlier, and just pass that. Potentially put flags on some kine of context?

Comment (T-Wizard) [reply apis/nucleus/src/components/listbox/ListBoxInline.jsx:105]: looking at how we done it before.
then we have had the use of flags in https://github.com/qlik-oss/sn-list-objects
and a parameter in nebula.js

but I am not sure why we did it that way

**Review by T-Wizard — APPROVED** [review]

File: apis/nucleus/src/components/listbox/ListBoxInline.jsx (hunk 1 above)
Comment (haxxmaxx) [reply apis/nucleus/src/components/listbox/ListBoxInline.jsx:105]: ok, I might have a look at that and revisit this. Would like to refactor a bit to align the code for selection colors in filterpane and in here

**Pull request merged**